service/src/main/java/org/cbioportal/genome_nexus/service/mixin/my_variant_info_mixin/ClinVarMixin.java

```diff
@@ -0,0 +1,41 @@
+package org.cbioportal.genome_nexus.service.mixin.my_variant_info_mixin;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import org.cbioportal.genome_nexus.model.my_variant_info_model.Gene;
+import org.cbioportal.genome_nexus.model.my_variant_info_model.Hg19;
+import org.cbioportal.genome_nexus.model.my_variant_info_model.Hg38;
+import org.cbioportal.genome_nexus.model.my_variant_info_model.Hgvs;
+
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+
+public class ClinVarMixin
```

Comment:
Indentation seems off in this class.

Reply:
@emz1003 indentation here seems to be off still, should be e.g.:
```
public class ClinVarMixin
{
   @JsonProperty(value = "license", required = true)
   private String license; 
}
```